packages/@webex/plugin-meetings/src/member/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface IExternalRoles {
+  cohost: boolean;
+  moderator: boolean;
+  presenter: boolean;
+}
+
+export enum ServerRoles {
+  Cohost = 'COHOST',
+  Moderator = 'MODERATOR',
+  Presenter = 'PRESENTER',
+}
+
+export type ServerRoleShape = {
```

Comment:
suggestion: Are you married to 'ServerRoleShape'?  It seems a little awkward to me.  Would this be better named 'ServerRole' and the other names 'ServerRoleType'?  Obviously minor, so certainly not a blocker.